### ISSUE TITLE
Crash after update my project with asp.net core 1.1

### ISSUE THREAD

**Issue by odinnou** [IdentityServer/IdentityServer3.AccessTokenValidation.Integration.AspNetCore#6]
Hi !
I work on a project which use IdentityServer 3 and I use IdentityServer3.AccessTokenValidation.Integration.AspNetCore to check if a Reference Token is allowed to access on actions decorated with [Authorize] attribute. Everything worked like a charm but...

Yesterday, I update my solution with the new version of Asp.Net Core (1.1) released the same day.
However during the execution I got this error :( 

[Internal Server Error.pdf](https://github.com/IdentityServer/IdentityServer3.AccessTokenValidation.Integration.AspNetCore/files/599328/Internal.Server.Error.pdf)

I try to use Identityserver4.AccessTokenValidation as a workaround solution, but I have always a 401 on every action decorate with [Authorize] attribute .

Thanks in advance!

**Comment by leastprivilege** [comment]
We haven't migrated to 1.1 yet. Consider yourself on the bleeding edge. Will have a look at it once again he rest of the ecosystem (e.g. Xunit) have caught up as well.

**Comment by leastprivilege** [comment]
Seems to be this problem

https://github.com/aspnet/Security/issues/1046#issuecomment-262235189

**Comment by odinnou** [comment]
Yeah :-/ It seems to be an issue with the v4.1 & v4.3 package System.net.http T_T
